### PR TITLE
plugins:foreman_docker - Release 1.4.1

### PR DESCRIPTION
**Pull request by eLobato**

### CODE REVIEW

File: plugins/ruby-foreman-docker/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-docker (1.4.1-1) stable; urgency=low
+
+  * Update release to 1.4.1
+
+ -- Daniel Lobato Garcia <elobatocs@gmail.com>  Tue, Aug 18 2015 10:42:21 +0200
```
Comment (domcleal) [inline plugins/ruby-foreman-docker/debian/changelog]: Date format's wrong again.

**Comment by domcleal** [comment]
Needs a rebase.

File: plugins/ruby-foreman-docker/debian/changelog (hunk 1 above)
Comment (domcleal) [inline plugins/ruby-foreman-docker/debian/changelog]: (Please use scripts/changelog.rb)

**Comment by eLobato** [comment]
[test] should work now that gem is in Rubygems.

**Comment by domcleal** [comment]
Thanks @eLobato, merged and building for nightlies.

**Pull request merged**